azure/services/virtualmachineimages/images.go

```diff
@@ -115,12 +115,14 @@ func (s *Service) GetDefaultWindowsImage(ctx context.Context, location, k8sVersi
 	return defaultImage, nil
 }
 
-// GetDefaultImageSKUID gets the SKU ID and version of the image to use for the provided version of Kubernetes.
+// getSKUAndVersion gets the SKU ID and version of the image to use for the provided version of Kubernetes.
```

Comment:
is this one meant to merge before or after #2388 ?

Reply:
Either way should be fine, they're independent (although rebases may happen). I'd like to see #2388 merge soon because it fixes a serious bug, but this just improves logging.